lib/twine/formatters/csv.rb

```diff
@@ -0,0 +1,64 @@
+require 'csv'
+
+module Twine
+  module Formatters
+    class CSV < Abstract
+      def format_name
+        'CSV'
+      end
+
+      def extension
+        '.csv'
+      end
+
+      def default_file_name
+        'strings.csv'
+      end
+
+      def determine_language_given_path(path)
+        path_arr = path.split(File::SEPARATOR)
+        path_arr.each do |segment|
+          match = /-(.+)\.csv$/.match(segment)
+          return match[1] if match
+        end
+
+        return
+      end
+
+      def read(io, lang)
+        while line = io.gets
+          fields = ::CSV.parse_line(line)
+
+          key = fields[0]
+          value = fields[1]
+          comment = fields[2]
+
+          # checks for nil, empty and whitespace
+          if key =~ /\S/ and value =~ /\S/
+            set_translation_for_key(key, lang, value)
+            set_comment_for_key(key, comment) if comment =~ /\S/
+          end
+        end
+      end
+
+      def format_header(lang)
+        'Key,Value,Comment'
+      end
+
+      def format_section(section, lang)
+        # removes empty lines
+        super.gsub(/^$\n/, '')
```

Comment:
If it's important to not have empty lines in the CSV, we should add a unit test for this